### PR TITLE
Create Abilities and UserAbilities schema

### PR DESCRIPTION
**Pull request by CDimonaco**
# Description

Create Abilities and UsersAbilities schema and context

## How was this tested?

Automated tests

### CODE REVIEW

File: lib/trento/abilities/users_abilities.ex [hunk 1]
```diff
@@ -0,0 +1,24 @@
+defmodule Trento.Abilities.UsersAbilities do
+  @moduledoc """
+  Many to main table schema used to connect users and abilities.
```
Comment (arbulu89) [inline lib/trento/abilities/users_abilities.ex:3]: `Many to main` -> `Many to many`.

And the next maybe better:
`We have a dedicated schema to implement the association between users and abilities, to just enable read only operation. Using the default ecto schema, declaring a user with a not existing ability would trigger the creation of this second, and we don't want to allow that. Abilities are just read only.

File: lib/trento/abilities/ability.ex [hunk 2]
```diff
@@ -0,0 +1,26 @@
+defmodule Trento.Abilities.Ability do
+  @moduledoc """
```
Comment (arbulu89) [inline lib/trento/abilities/ability.ex:2]: Let's add a docstring or put `@moduledoc false`.
I would maybe document that the `label` is just an informative field to describe what the ability does

File: lib/trento/abilities.ex [hunk 3]
```diff
@@ -0,0 +1,104 @@
+defmodule Trento.Abilities do
```
Comment (arbulu89) [inline lib/trento/abilities.ex:1]: Thank you phoenix :rocket:

**Review by arbulu89 — APPROVED** [review]
Perfect! So cool

File: lib/trento/abilities/users_abilities.ex (hunk 1 above)
Comment (CDimonaco) [reply lib/trento/abilities/users_abilities.ex]: done

**Pull request merged**